mmv1/templates/terraform/resource.html.markdown.tmpl

```diff
@@ -58,6 +58,11 @@ To get more information about {{$.Name}}, see:
 values will be stored in the raw state as plain text: {{ $.SensitivePropsToString }}.
 [Read more about sensitive data in state](https://www.terraform.io/language/state/sensitive-data).
 {{ end }}
+{{- if $.WriteOnlyProps }}
+~> **Warning:** All arguments including the following potentially write-only
+values will be stored in the raw state as plain text: {{ $.WriteOnlyPropsToString }}.
```

Comment:
Isn't the whole point that they aren't stored in state? What am I missing here?

Reply:
this is wrong and has been addressed in the latest commit.